src/import-data/import-utils.feature-flag.test.ts

```diff
@@ -0,0 +1,23 @@
+import { ImportRequest } from './import-types';
+import { getCloudPlatformRequiredForImport } from './import-utils';
+
+jest.mock('src/libs/feature-previews', () => ({
+  ...jest.requireActual('src/libs/feature-previews'),
```

Comment:
It would be nice to add types for this mock.

```suggestion
type FeaturePreviewExports = typeof import('src/libs/feature-previews');
jest.mock('src/libs/feature-previews', (): FeaturePreviewExports => ({
  ...jest.requireActual<FeaturePreviewExports>('src/libs/feature-previews'),
```